app/views/project_content_items/update_flags.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#flags-modal').modal('toggle');
+$('.content-item form[data-ref="<%= content_item.id %>"]').parent('.content-item').fadeOut('slow');
```

Comment:
Same question as a [previous comment](), should we also remove the element from the page when it's visually hidden?

Reply:
Also not to be too picky, but to be consistent with the animation style, maybe this can fade out fast to match what happens when you press "Done".

Reply:
I played around with different animation speeds, and I think the slower fade here works well. There's a few things happening onscreen, and if this fades too fast then it's easy to miss what's happened while the modal closes.